homeassistant/components/nuki/__init__.py

```diff
@@ -13,7 +13,7 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_HOST, CONF_PORT, CONF_TOKEN, Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers import entity_registry as er
+from homeassistant.helpers import device_registry as dr, entity_registry as er
```

Comment:
This shortening makes it hard to read. Let's move forward with `device_registry` and `entity_registry`

Reply:
removed the shortenings